scan-irb/report-en.tex

```diff
@@ -58,7 +58,23 @@
 \begin{description}[font=\normalfont,align=left,labelwidth=12em]
 \item [\textbf{\VAR{pat_name|e}}]
 \item [\textbf{Date of Birth}] \VAR{birth_date|e}
-\item [\textbf{Specimen}] Midnasal, self-collected
+\item [\textbf{Specimen}]
+  %- if swab_type == "mts"
+  Midnasal,
+  %- elif swab_type == "ans"
+  Anterior,
+  %- else
+  \errmessage{Invalid swab_type "\VAR{swab_type}"}
+  %- endif
```

Comment:
The current values in the specimen manifest dropdown menu are `mtb`, `ans`, `np`, and `unk`.

We should definitely error out for `unk` but we should include the other values here.

Reply:
Thanks for this info!